python/tests/test_xray.py

```diff
@@ -634,3 +636,72 @@ def test_ionchamber_fluxes():
 
     assert_allclose(ic4.transmitted/ic4.incident, 0.4928, rtol=0.01)
     assert_allclose(ic4.incident, 2.7615e10, rtol=0.01)
+
+
+def test_formula_to_mass_fracs():
+    mf1 = formula_to_mass_fracs('Fe2O3')
+    desired1 = {'Fe': 0.69943, 'O': 0.30056}
+    assert_allclose(mf1['Fe'], desired1['Fe'], rtol=0.001)
+    assert_allclose(mf1['O'], desired1['O'], rtol=0.001)
+    assert_allclose(sum(mf1.values()), 1.0, rtol=0.001)
+    
+    mf2 = formula_to_mass_fracs({'Fe2O3':1, 'FeO':2})
+    desired2 = {'Fe': 0.7363, 'O': 0.2637}
+    assert_allclose(mf2['Fe'], desired2['Fe'], rtol=0.001)
+    assert_allclose(mf2['O'], desired2['O'], rtol=0.001)
+    assert_allclose(sum(mf2.values()), 1.0, rtol=0.001)
+
+
+def test_validate_mass_fracs():
```

Comment:
I can confirm that this test fails in master.
```
    def test_validate_mass_fracs():
        mf = _validate_mass_fracs({'Fe2O3': 0.75, 'FeO': 0.25})
        desired = {'Fe': 0.7189, 'O': 0.2811}
>       assert_allclose(mf['Fe'], desired['Fe'], rtol=0.001)
E       AssertionError: 
E       Not equal to tolerance rtol=0.001, atol=0
E       
E       Mismatched elements: 1 / 1 (100%)
E       Max absolute difference: 0.52457271
E       Max relative difference: 0.72968801
E        x: array(0.194327)
E        y: array(0.7189)
```

Reply:
Sure thing! Ah excellent I hadn't gone that far to double check the test in the master branch, thanks!